### PR TITLE
Implementation/syntax highlight

### LINKED ISSUE #24: Display student code submissions in Instructor Dashboard
If possible with syntax highlighting :)

### PR DESCRIPTION
**Pull request by cicerotcv**
Fix #24

### CODE REVIEW

File: frontend/frontend-dev/src/components/CodeVisualizer/index.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ export function CodeVisualizer({
         wrapLines={false}
         lineNumberStyle={{
           fontWeight: "bold",
-          color: "#c0c0c050",
+          color: "##555a7b",
```
Comment (toshikurauchi) [inline frontend/frontend-dev/src/components/CodeVisualizer/index.tsx:24]: I think there's an extra `#` here

File: frontend/frontend-dev/src/fragments/ExerciseResultVisualization/InstructorCodeExerciseResult/index.tsx [hunk 2]
```diff
@@ -1,9 +1,10 @@
-import { useState } from "react";
+import { Fragment, useState } from "react";
```
Comment (toshikurauchi) [inline frontend/frontend-dev/src/fragments/ExerciseResultVisualization/InstructorCodeExerciseResult/index.tsx:1]: Do we need to import `Fragment` here?

File: frontend/frontend-dev/src/fragments/ExerciseResultVisualization/InstructorCodeExerciseResult/index.tsx [hunk 3]
```diff
@@ -48,11 +72,32 @@ export function CodeExerciseResult({ codeAnswers }: ICodeResultProps) {
...
+                      className="m-0 p-0 mx-1 align-middle"
+                      value={codePk}
+                      checked={itemIsComparing}
+                      onChange={() => {
+                        selectToCompare(codePk, codeText);
+                      }}
+                    />
+                  </label>
+                ),
+                // compare: (
```
Comment (toshikurauchi) [inline frontend/frontend-dev/src/fragments/ExerciseResultVisualization/InstructorCodeExerciseResult/index.tsx:94]: Do we need this comment?

**Review by toshikurauchi — CHANGES_REQUESTED** [review]
Just a couple of comments, but everything looks good.

In the storybook setting values to `leftTitle`, `rightTitle` and `title` didn't do anything, is that expected?

**Comment by cicerotcv** [comment]
> Just a couple of comments, but everything looks good.
> 
> In the storybook setting values to `leftTitle`, `rightTitle` and `title` didn't do anything, is that expected?

That's right. It's not implemented yet. It will work when the header of the CodeBlock component. I'll fix this PR addressing the comments you've made and removing these props for now.

**Comment by toshikurauchi** [comment]
> > Just a couple of comments, but everything looks good.
> > In the storybook setting values to `leftTitle`, `rightTitle` and `title` didn't do anything, is that expected?
> 
> That's right. It's not implemented yet. It will work when the header of the CodeBlock component. I'll fix this PR addressing the comments you've made and removing these props for now.

You can keep the props, it's fine. I just wanted to confirm that they were not being used yet. Let me know when you want me to re-review your PR.

**Review by toshikurauchi — APPROVED** [review]

**Pull request merged**